homeassistant/components/axion_dmx/config_flow.py

```diff
@@ -0,0 +1,72 @@
+"""Config flow for Axion Lighting integration."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from libaxion_dmx import AxionDmxApi, AxionDmxAuthError, AxionDmxConnectionError
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_HOST, CONF_PASSWORD
+from homeassistant.helpers import selector
+
+from .const import _LOGGER, CONF_CHANNEL, CONF_LIGHT_TYPE, DOMAIN
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): str,
+        vol.Required(CONF_PASSWORD): str,
+        vol.Required(CONF_CHANNEL): vol.All(vol.Coerce(int), vol.Range(min=1)),
+        vol.Required(CONF_LIGHT_TYPE): selector.SelectSelector(
+            selector.SelectSelectorConfig(
+                options=[
+                    selector.SelectOptionDict(value="white", label="White"),
+                    selector.SelectOptionDict(
+                        value="tunable_white", label="Tunable White"
+                    ),
+                    selector.SelectOptionDict(value="rgb", label="RGB"),
+                    selector.SelectOptionDict(value="rgbw", label="RGBW"),
+                    selector.SelectOptionDict(value="rgbww", label="RGBWW"),
+                ],
+                translation_key="light_type",
+                multiple=False,
+                mode=selector.SelectSelectorMode.DROPDOWN,
+            )
+        ),
+    }
+)
+
+
+class AxionConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Axion Lighting."""
+
+    VERSION = 1
+
+    async def async_step_user(
```

Comment:
The user flow is not aborting if we try to setup the same device again, https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/unique-config-entry